index.js

```diff
@@ -84,10 +85,47 @@ Lexer.prototype = {
    * @api private
    */
 
-  tok: function(type, val){
-    return val === undefined ?
-      {type: type, line: this.lineno} :
-      {type: type, line: this.lineno, val: val};
+  tok: function(type, val, columnIncrement){
+    var res = {type: type, line: this.lineno};
+
+    if (this.colno !== null) res.col = this.colno;
```

Comment:
This was useful during development so that only certain tokens in the test cases updated, but yes I agree it's unneeded now fully implemented, I'll chance this